src/pieces/messageCount.ts

```diff
@@ -17,6 +17,10 @@ async function register(bot: Client): Promise<void> {
 	bot.on('messageCreate', async msg => {
 		countMessages(msg).catch(async error => bot.emit('error', error));
 	});
+	bot.on('messageDelete', async msg => {
+		if (msg.content && msg.content.startsWith(';s')) return;
```

Comment:
`msg.content.startsWith(';s')` ?? 
```suggestion
		if (msg.content && msg.content.startsWith('s;')) return;
```

Reply:
I get that this doesn't matter after slash commands bc they don't get registered as messages, but still good to have working code

Reply:
unintentional typo, good call there